packages/editor/src/components/common/dropdown/dropdownContainer.tsx

```diff
@@ -92,7 +92,7 @@ export class DropdownContainer extends React.Component<IProps, IState> {
                 <DropdownBasic
                     outsideClickIgnoreClass={this.ignoreClassName}
                     handleClickOutside={this.closeMenu}
-                    handleClickInside={!enableClickInside ? this.closeMenu : undefined}
+                    // handleClickInside={!enableClickInside ? this.closeMenu : undefined}
```

Comment:
If it's not commented out, it will close the `Select an account` window immediately if you click inside of it, we do not want that anymore since we are changing the name inside of it so I think this line can be removed all together.

Reply:
Ahh but this is a common component, so it will probably affect other parts of the code if it's commented out :( do you have an idea of what we can do to prevent it on our `Select an account` window only?